Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -27,7 +28,7 @@ public ACCOUNT GetAccountByParticipantID(int ID)
             return _context.ACCOUNT
                         .FirstOrDefault(a => a.gordon_id == $"{ID}");
         }
-        private int GetParticipantID(string username)
+        private int GetParticipantIdByUsername(string username)
         {
             return Int32.Parse(_context.ACCOUNT
                         .FirstOrDefault(a => a.AD_Username == username)
```

Comment:
I might suggest using the 360 account service rather thank your own since it already has this functionality.